### PR TITLE
Improved bitmap docs

### PR DESCRIPTION
**Pull request by GumpacG**

### CODE REVIEW

File: java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java [hunk 1]
```diff
@@ -289,10 +289,11 @@ CompletableFuture<Long> bitop(
 
     /**
      * Reads the array of bits representing the string that is held at <code>key</code> based on the
-     * specified <code>subCommands</code>.
+     * specified <code>subCommands</code>. Can be routed to read-only replicas to allow for <code>
+     * BitField</code> behaviour in read-only replicas.
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:293]: The phrase is correct and not at the same time.
We do not provide routing option for keyed commands. Should we add it here? Routing is determined automatically by GLIDE and user can do nothing there. A notice is correct, but useless.

Comment (GumpacG) [reply java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:293]: Should it just be `allows for <code>BitField</code> behaviour in read-only replicas.`?

Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:293]: As discussed with @GilboaAWS, we need to copy it from #1528
> This command is routed depending on the the client's `ReadFrom` strategy.

Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:293]: Add `{@link ..}` to this setting.

Comment (GumpacG) [reply java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java]: Updated. Thanks!

File: java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java [hunk 2]
```diff
@@ -289,10 +290,11 @@ CompletableFuture<Long> bitop(
 
     /**
      * Reads the array of bits representing the string that is held at <code>key</code> based on the
-     * specified <code>subCommands</code>.
+     * specified <code>subCommands</code>. This command is routed depending on the client's {@link
+     * ReadFrom} strategy.
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:294]:
```suggestion
     * specified <code>subCommands</code>.<br>
     * This command is routed depending on the client's {@link ReadFrom} strategy.
```

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 3]
```diff
@@ -4092,10 +4093,11 @@ public T bitfield(@NonNull String key, @NonNull BitFieldSubCommands[] subCommand
 
     /**
      * Reads the array of bits representing the string that is held at <code>key</code> based on the
-     * specified <code>subCommands</code>.
+     * specified <code>subCommands</code>. This command is routed depending on the client's {@link
+     * ReadFrom} strategy.
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:4097]:
```suggestion
     * specified <code>subCommands</code>.<br>
     * This command is routed depending on the client's {@link ReadFrom} strategy.
```

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**